pause.go

```diff
@@ -0,0 +1,46 @@
+//
+// Copyright (c) 2017 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package main
+
+/*
+#cgo CFLAGS: -Wall
+#define _GNU_SOURCE
+#include <signal.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+
+#define PAUSE_BIN "pause-bin"
+
+static void sigdown(int signo) {
+	psignal(signo, "shutting down, got signal");
+	exit(0);
+}
+
+void __attribute__((constructor)) constructor_map(int argc, const char **argv) {
```

Comment:
The example code use "constructor_map()" to show the memory map on the stage of "__attribute__((constructor))".

And the proper name here might be "sandbox_init_process()", or "sandbox_pause()", or something else...

Reply:
Yes, sorry I forgot to change the name...

Reply:
done